plugins/mem0-ops/scripts/doctor.py

```
@@ -0,0 +1,92 @@
+        except (json.JSONDecodeError, OSError):
+            pass
+    check(
+        "auto_save",
+        auto_save is False,
+        f"~/.mem0/settings.json auto_save={auto_save} "
+        f"(this file governs; env MEM0_AUTO_SAVE is overwritten per hook)",
+    )
+
+    try:
+        decay = req_json(f"{BASE}/api/v1/orgs/projects/?fields=decay")
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)